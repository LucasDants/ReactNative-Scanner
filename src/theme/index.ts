export default {
  colors: {
    red: {
      900: '#B83341',
      800: '#E03F50',
    },
    blue: '#0000FF',
    white: '#FFFFFF',
    gray: {
      100: '#DCDCDC',
      600: '#404040',
      900: '#030303',
    },
  },
};