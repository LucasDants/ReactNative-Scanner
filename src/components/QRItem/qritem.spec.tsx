import React from 'react'

import { render } from '@testing-library/react-native'

import { ThemeProvider } from 'styled-components/native'
import theme from '../../theme'

import { QRItem } from '.'

const Providers = ({ children }) => (
  <ThemeProvider theme={theme}>
    { children }
  </ThemeProvider>
)

describe('QRItem Component', () => {

  it("should match the snapshot test", () => {
    const { toJSON } = render(
      <QRItem>Test</QRItem>, {wrapper: Providers}
    )

    const component = toJSON()
    expect(component).toMatchSnapshot()

  })

  it("should have the correct children", () => {
    const { getByText } = render(
       <QRItem>Test</QRItem>, {wrapper: Providers}
    )

    expect(getByText('Test')).toBeTruthy()

  })

})