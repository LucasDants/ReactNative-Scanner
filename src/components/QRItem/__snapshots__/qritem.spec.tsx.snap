// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRItem Component should match the snapshot test 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 24,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 24,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 27,
        },
      ]
    }
  >
    Test
  </Text>
</View>
`;
